### PR TITLE
Fe25519 V1.5.0

### PR DESCRIPTION
**Pull request by Arishamays1**
# ☆ Fe25519 Update ☆
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-3118)
[Upstream](https://github.com/nthparty/fe25519)
[Dependencies ](https://github.com/nthparty/fe25519/blob/1.5.0/pyproject.toml)

## Changes
- Newly rendered `meta.yaml` added to `recipe`
- Added information to `about` section
- Pinning and dependency amendments/formatting

## Relates to:
#https://github.com/AnacondaRecipes/oblivious-feedstock/pull/1
#https://github.com/AnacondaRecipes/ge25519-feedstock/pull/1

### CODE REVIEW

**Review by boldorider4 — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
Is there the missing `abs.yaml` with the destination channel `sfe1ed40`?

**Comment by boldorider4** [comment]
> Is there the missing `abs.yaml` with the destination channel `sfe1ed40`?

good point

**Review by boldorider4 — DISMISSED** [review]
`abs.yaml` is missing, otherwise LGTM.

**Review by boldorider4 — DISMISSED** [review]

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "fe25519" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fe25519-{{ version }}.tar.gz
```
Comment (boldorider4) [inline recipe/meta.yaml]: `fe25519` should be replaced with `{{ name }}`

**Comment by Arishamays1** [comment]
> Is there the missing `abs.yaml` with the destination channel `sfe1ed40`?

Yes Thank you all for adding it. I needed to confirm that it will be uploaded to Snowflake, so I hadn't added it quite yet!

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Good eye!

**Pull request merged**